### ISSUE TITLE
missing some files 

### ISSUE THREAD

**Issue by SeekPoint** [donglixp/coarse2fine#21]
mldl@ub1604:/ub16_prj/coarse2fine$ bash run.sh atis 0
Preparing training ...
Building Training...
Building Valid...
Building Test...
Building Vocab...
Saving train/valid/fields
run.sh: line 23: 25247 Segmentation fault (core dumped) CUDA_VISIBLE_DEVICES=$GPU_ID python3 train.py -root_dir "$WORK_DIR/data_model/" -dataset $DATANAME -rnn_size 250 -word_vec_size 200 -decoder_input_size 150 -layers 1 -start_checkpoint_at 60 -learning_rate 0.005 -start_decay_at 0 -epochs 100 -global_attention "dot" -attn_hidden 0 -lock_dropout -dropout 0.5 -dropout_i 0.5
head: cannot open '/home/mldl/ub16_prj/coarse2fine/data_model/atis/dev_best.txt' for reading: No such file or directory
mldl@ub1604:/ub16_prj/coarse2fine$

**Comment by donglixp** [comment]
Hi @loveJasmine ,

The error is caused by "Segmentation fault". Could you try to use "CUDA_LAUNCH_BLOCKING=1,CUDA_VISIBLE_DEVICES=$GPU_ID" before the command "python triain.py" for debugging? Thanks!

Best,
Li

**Comment by SeekPoint** [comment]
mldl@ub1604:~/ub16_prj/coarse2fine$ bash run.sh atis 0
Preparing training ...
Building Training...
Building Valid...
Building Test...
Building Vocab...
Saving train/valid/fields
run.sh: line 23:  9831 Segmentation fault      (core dumped) CUDA_VISIBLE_DEVICES=$GPU_ID python3 train.py -root_dir "$WORK_DIR/data_model/" -dataset $DATANAME -rnn_size 250 -word_vec_size 200 -decoder_input_size 150 -layers 1 -start_checkpoint_at 60 -learning_rate 0.005 -start_decay_at 0 -epochs 100 -global_attention "dot" -attn_hidden 0 -lock_dropout -dropout 0.5 -dropout_i 0.5
head: cannot open '/home/mldl/ub16_prj/coarse2fine/data_model/atis/dev_best.txt' for reading: No such file or directory
mldl@ub1604:~/ub16_prj/coarse2fine$

**Comment by lodgegao** [comment]
Hello Li, I tried using "CUDA_LAUNCH_BLOCKING=1" but the problem exists still.

(py35) lodgegao@localhost:/home/lodgegao/c2f$ ./run.sh django 2
Preparing training ...
Building Training...
Building Valid...
Building Test...
Building Vocab...
Saving train/valid/fields
./run.sh: line 32: 11726 Segmentation fault      CUDA_LAUNCH_BLOCKING=1 CUDA_VISIBLE_DEVICES=$GPU_ID python train.py -root_dir "$WORK_DIR/data_model/" -dataset $DATANAME -rnn_size 300 -word_vec_size 250 -decoder_input_size 200 -layers 1 -start_checkpoint_at 15 -learning_rate 0.002 -epochs 25 -global_attention "dot" -attn_hidden 0 -dropout 0.3 -dropout_i 0.3 -lock_dropout -copy_prb hidden
head: cannot open ‘/home/lodgegao/c2f/data_model/django/dev_best.txt’ for reading: No such file or directory

Is there anything I did wrong? Hope for your help.

**Comment by donglixp** [comment]
There is still "Segmentation fault". The python debugger "pudb" (https://documen.tician.de/pudb/) could be used to find where the code is stopped.

Best,
Li

**Comment by lodgegao** [comment]
@donglixp 
Hi Li
I found the problem occurs in the line "from tensorboard_logger import Logger", and the module Logger is not used in the following code, so I commented out that line, the the problem gets solved. Thank you.

Sincerely
Gao

**Comment by donglixp** [comment]
Hi @lodgegao ,

I used "tensorboard_logger==0.0.4" to log the experiments. It might contain some bugs to import the package in your environment. Thanks for letting me know.

Best,
Li
